src/control-plane/cloudfront-s3-portal.ts

```diff
@@ -188,6 +189,9 @@ export class CloudFrontS3Portal extends Construct {
       prune: false,
       distribution: this.distribution,
       distributionPaths: props.frontendProps.autoInvalidFilePaths,
+      cacheControl: [
+        CacheControl.maxAge(Duration.days(1)),
```

Comment:
Will it set the cache control of all files to 1 day?

If so, I prefer to create another BucketDeployment instance to upload `index.html` only with 1-day cache control.